### DISCUSSION TITLE
Does Bromite's adblocker support cosmetic filtering?

### DISCUSSION THREAD

**Discussion by dbui04** [bromite/bromite#1214] [Q&A] [answered]
If it does, can you consider adding some annoyance filter lists to the default block list?

**Comment by chromer030** [comment, marked as answer]
You can use this unofficial filter : https://github.com/xarantolus/filtrite , while Bromite has by default 8 list of filters , bromite-extended has 20 list of filters and because of automatic list generation it has very up-to-date filters , also you can request for filters in that repo.

**Reply by dbui04** [reply]
Thank you for that. I have found my answer: The wiki indicated that CSS rules weren't supported, therefore cosmetic filtering is currently not possible. Hope the devs will make it work in the future :)
